IceCubesApp/App/Tabs/Settings/ContentSettingsView.swift

```diff
@@ -19,6 +19,12 @@ struct ContentSettingsView: View {
         }
       }.listRowBackground(theme.primaryBackgroundColor)
 
+      Section("settings.content.media") {
+        Toggle(isOn: $userPreferences.showAltTextForMedia) {
+         Text("settings.content.media.show.alt")
+        }
+      }
```

Comment:
Missing the listRowBackground.

Reply:
fixed and pushed. Seems I need to run the simulator through different themes to see this.